packaging/rpm/SPECS/nvidia-container-toolkit.spec

```diff
@@ -89,6 +103,9 @@ Provides tools such as the NVIDIA Container Runtime and NVIDIA Container Toolkit
 %{_bindir}/nvidia-container-runtime
 %{_bindir}/nvidia-ctk
 %{_bindir}/nvidia-cdi-hook
+%dir /etc/systemd/system
```

Comment:
Surely this is not correct? This directory is not part of the package. Why do we not create this folder when we install the files?

Reply:
yes, you are right, removed